packages/components/src/tooltip/stories/index.story.tsx

```diff
@@ -57,3 +57,14 @@ KeyboardShortcut.args = {
 		ariaLabel: shortcutAriaLabel.primaryShift( ',' ),
 	},
 };
+
+// TODO: remove before merging
```

Comment:
This example is only around for ease of reviewing / testing. I will remove it before merging.

Reply:
[`1b7f1a1` (#57202)](https://github.com/WordPress/gutenberg/pull/57202/commits/1b7f1a12606506bf48878ef3b2bbe9a930a313c1)